services/dask-sidecar/tests/unit/test_utils_dask.py

```diff
@@ -117,11 +120,10 @@ async def _dask_sub_consumer_task(sub: distributed.Sub) -> None:
 
     async def _dask_publisher_task(pub: distributed.Pub) -> None:
         print("--> starting publisher task")
-        for n in range(NUMBER_OF_MESSAGES):
-            event_to_publish = TaskLogEvent(
+        for _ in range(NUMBER_OF_MESSAGES):
+            event_to_publish = TaskProgressEvent(
                 job_id=job_id,
-                log=f"the log {n}",
-                log_level=logging.INFO,
+                progress=0.5,
```

Comment:
Q: is this correct to be fixed?

Reply:
yes. the test runs now against the progress event log which will be removed in the next steps